core/src/models/App.ts

```diff
@@ -92,12 +92,9 @@ export class App extends LoggedModel<App> {
         ? await pluginApp.methods.appOptions()
         : {};
 
-    for (const o of staticAppOptions) {
-      if (o.type) {
-        if (!appOptions[o.key]) appOptions[o.key] = { type: o.type };
-        if (!appOptions[o.key].type) {
-          appOptions[o.key].type = staticAppOptions[o.key].type;
-        }
```

Comment:
I don't think this would have ever actually matched

Reply:
`noImplicitAny` and Typescript doing work 💪